clients/java/src/main/java/io/camunda/client/impl/fetch/DecisionInstanceGetRequestImpl.java

```diff
@@ -53,7 +55,7 @@ public FinalCommandStep<DecisionInstance> requestTimeout(final Duration requestT
   public CamundaFuture<DecisionInstance> send() {
     final HttpCamundaFuture<DecisionInstance> result = new HttpCamundaFuture<>();
     httpClient.get(
-        String.format("/decision-instances/%s", decisionInstanceId),
+        String.format("/decision-instances/%s", decisionEvaluationInstanceKey),
```

Comment:
❓ I assume we keep `decision-instances` and are not moving to `decision-evaluation-instances`

Reply:
I remember we decided to keep the "old" path, @aleksander-dytko can you confirm this ?